.github/workflows/yarn-and-test-cli.yml

```diff
@@ -1,14 +1,10 @@
-name: Yarn Install and Test JS
+name: CandyMachine: Yarn Install and Test JS
```

Comment:
This is not specific to CandyMachine, why the name change?

Reply:
cli is the candy machine